keras_cv/losses/focal_test.py

```diff
@@ -68,13 +57,26 @@ def test_output_shape_from_logits(self):
         )
 
     def test_from_logits_argument(self):
-        y_true = tf.random.uniform((2, 8, 10))
-        y_logits = tf.random.uniform((2, 8, 10), minval=-1000, maxval=1000)
-        y_pred = tf.nn.sigmoid(y_logits)
+        rng = np.random.default_rng(1337)
+        y_true = rng.uniform(size=(2, 8, 10)).astype("float64")
+        y_logits = rng.uniform(low=-1000, high=1000, size=(2, 8, 10)).astype(
+            "float64"
+        )
+        y_pred = ops.cast(ops.sigmoid(y_logits), "float32")
 
         focal_loss_on_logits = FocalLoss(from_logits=True)
         focal_loss = FocalLoss()
 
+        # TODO(ianstenbit): This probably warrants some more investigation.
```

Comment:
Is it just the loss scale that differs or also the performance metrics?

Reply:
The performance metrics match when I train the model with different backends -- I think the root cause here is op fusion of softmax + xent, but I'm not certain yet (hence leaving the TODO)